askama/Cargo.toml

```diff
@@ -33,6 +33,7 @@ with-mendes = ["askama_derive/with-mendes"]
 with-rocket = ["askama_derive/with-rocket"]
 with-tide = ["askama_derive/with-tide"]
 with-warp = ["askama_derive/with-warp"]
+i18n = ["askama_derive/i18n", "fluent-templates"]
```

Comment:
as this is a `fluid` backed i18n, would it be appropriate to solf-document that in the feature name? e.g. `i18n_fluent`?